.github/workflows/build_and_test.yml

```diff
@@ -273,29 +273,27 @@ jobs:
           - ./fuzzers/binary-only/frida_executable_libpng
           - ./fuzzers/binary-only/frida_gdiplus
           - ./fuzzers/binary-only/frida_libpng
+          - ./fuzzers/binary-only/fuzzbench_qemu
           - ./fuzzers/binary-only/tinyinst_simple
 
           # Forkserver
           - ./fuzzers/forkserver/forkserver_simple
           - ./fuzzers/forkserver/forkserver_libafl_cc
+          - ./fuzzers/forkserver/fuzzbench_fork_qemu
```

Comment:
That guy is probably also binary-only (?)

Reply:
yes it is, i don't know which should be used

Reply:
I would say it's more important that it's binary only then
Like, full system > binary only > inprocess/forkserver

Reply:
from more to less specific so, i'm fine with this